src/systemcmds/pwm/pwm.c

```diff
@@ -448,6 +449,50 @@ pwm_main(int argc, char *argv[])
 
 		exit(0);
 
+//	} else if (!strcmp(argv[1], "trim")) {
```

Comment:
this code is commented out, but should be removed or (somehow) modified to handle signed normalized values for "p"
The original functionality is replaced by the trim parameter handling.